html/infrastructure/urls/dynamic-changes-to-base-urls/dynamic-urls.sub.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>HTML Test: Dynamic changes to base URLs</title>
+<link rel="author" title="Intel" href="http://www.intel.com/"/>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#dynamic-changes-to-base-urls" />
+<base href="" id="base">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<div id="div" style="display:none"></div>
+<script>
+      var div = document.getElementById("div"),
+          base = document.getElementById("base"),
+          url =  document.location.href;
+
+      var testData = [
+        {elements: ["a", "link", "area"], set: "href", get: "href"},
+        {elements: ["a", "area"], set: "ping", get: "ping"},
```

Comment:
I ran the tests in Chrome and Firefox, and the ping tests fail in both. (All other tests pass.)

That's because the [ping attribute](https://html.spec.whatwg.org/multipage/links.html#ping) is a space-separated list of URLs, not a single URL. So the reflection is verbatim. The base URL presumably still has some effect for ping URLs, but it's not possible to test it just by reflection. I'd suggest just removing this line.

Reply:
@foolip, thanks for your comments, I've addressed it in new commit, PTAL.